### PR TITLE
Add sleep timer to HO test

### PR DESCRIPTION
**Pull request by trozet**
There's a race between when the namespace is updated and the new pod
that gets created right after. The problem is nsInfo in the ovn-k8s
backend may not be updated (has to wait for lock) by the time the new
pod gets created. A 1 second sleep fixes the issue.

Signed-off-by: Tim Rozet <trozet@redhat.com>

### CODE REVIEW

**Comment by trozet** [comment]
@girishmg @dcbw FYI

**Comment by trozet** [comment]
@girishmg I think you tried this and you were not able to reproduce the failure anymore correct?

**Comment by girishmg** [comment]
@trozet That is correct.

**Comment by girishmg** [comment]
@trozet let me try again

**Comment by girishmg** [comment]
@trozet don't see those failures anymore.

**Comment by girishmg** [comment]
/lgtm

I am going to merge this since at the very least making the `make check` pass for now.
